src/lib/core/mp_utils.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+/*
+ * SPDX-License-Identifier: BSD-2-Clause
+ *
+ * Copyright 2021, Tarantool AUTHORS, please see AUTHORS file.
+ */
+
+#include <limits.h>
+#include <assert.h>
+
+#include "msgpuck.h"
+
+/**
+ * Convenient wrapper for determining size of integer value
+ * in MessagePack, regardless of it's sign.
+ *
+ * @param n signed 64-bit value
+ * @sa mp_sizeof_int
+ * @sa mp_sizeof_uint
+ */
+static inline uint32_t
+mp_sizeof_xint(int64_t n)
+{
+	return n < 0 ? mp_sizeof_int(n) : mp_sizeof_uint(n);
+}
+
+/**
+ * Convenient wrapper for encoding integer value
+ * to messagepack, regardless of it's sign.
+ *
+ * @param data output MessagePack buffer
+ * @param v 64-bit value to be encoded.
+ * @sa mp_encode_int
+ * @sa mp_encode_uint
+ */
+static inline char *
+mp_encode_xint(char *data, int64_t v)
```

Comment:
Nothing in this file is ever used anywhere. Please, drop it. These funcs are not tested and won't work in the most painful place - SQL. Where we need to support `[INT64_MIN, UINT64_MAX]` range. So having just `int64_t` variable is not enough anyway.

Reply:
It used to be used in prior encoding schema. (And used in the different PR with encoding benchmark). Now the fastest, and fattest schema won, so it became redundant. Will remove.